cli/spinner.ts

```diff
@@ -81,6 +83,12 @@ export interface SpinnerOptions {
    * This can be changed while the spinner is active.
    */
   color?: Color;
+  /**
+   * Stream to write the spinner to.
+   *
+   * @default {Deno.stderr}
+   */
+  stream?: WriterSync & { writable: WritableStream<Uint8Array> };
```

Comment:
How about just accepting `"stderr" | "stdout"`? Dependency to `std/io` doesn't seem ideal.

Reply:
I was thinking of instead changing it to `writable?: WritableStream<Uint8Array>` with a default value of `Deno.stderr.writable`. That way, that'd make it much more flexible and drop the `@std/io` dependency.

Reply:
While my above suggestion doesn't completely eliminate the tie to the Deno runtime (we use `Deno.noColor`), it at least reduces it, making future compatibility with other runtimes a little easier.